Assignment/solution_student/index.html

```diff
@@ -1,35 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+    <title>Joe Johnson</title>
+    <link rel="stylesheet" href="style.css">
+</head>
 
-Sample Content
-——-
+<header>
 
-ABOUT ME PAGE
+<h2><em>About Me</em> | <a href="resume.html">Porfolio</a></h2>
+<h1>Joe Johnson</h1>
 
-I ENJOY LIFE AS A DEVELOPER
 
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
+</header>
 
+<body>
+<hr>
 
-RESUME PAGE
+<div id="bio">
+		<h3>I Enjoy Life as Developer</h3>
 
-Experience
-        
-I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.
-    
-Relevant Experience
-    
-Web Developer, Relaxr 2015 - present
-Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.
+		<p>I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.</p>
+	</div>
+<div id="photo">
+		<img src="images/avatar.png" alt="Joe Johnson Photo" />
+	</div>
+</div>
 
-Web Developer, Startup Matchmaker 2015 - present
-Used a design team's wireframes to develop this company's responsive homepage.
+<div style="clear: both;"></div>
 
-Web Developer, CitiPix 2015 – present
-Built web app prototype that allows users to store and quickly retrieve photos using keywords.
 
+<footer>
+<hr>
 
-EDUCATION
-
-General Assembly — FEWD 2015
-College of Soft Knocks — B.S. 2009
-School of Hard Knocks — 2005 (Honors)
+<p><b><a href="http://facebook.com">Facebook</a> | <a href="http://twitter.com">Twitter</a> | <a href="http://instagram.com">Instagram</a> | <a href="http://linkedin.com">LinkedIn</a></b></p>
 
+</footer>
```

Comment:
Oh no! You forgot the closing tags of the document! Every document should end with: 
```
  </body>
</html>
```
Remember, **everything** is wrapped in the `<html>` tag, and everything _visible_ on the site is wrapped in the `<body>` tag.